functions/graphql/schema/flightRecord.js

```diff
@@ -38,7 +38,7 @@ module.exports.schema = gql`
 
   # Get all the flight records that are tied to this particular badge (meaning that they have a station assigned to that badge)
   extend type Badge {
-    users: [User]
+    users: [User] @auth(requires: [directory])
```

Comment:
`directory` isn't a permission ;)
```suggestion
    users: [User] @auth(requires: [director])
```